src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -45,7 +43,7 @@ const Conversation: React.FC<ConversationProps> = props => {
     }
   }
 
-  useEffect(scrollToBottom, [conversation])
+  useEffect(scrollToBottom, [conversation, lastMessageID])
```

Comment:
A little eslint warning about not tracking a dependency of the `useEffect` hook.